do.sh

```diff
@@ -34,11 +34,39 @@ ovn_tester=${topdir}/ovn-tester
 EXTRA_OPTIMIZE=${EXTRA_OPTIMIZE:-no}
 USE_OVSDB_ETCD=${USE_OVSDB_ETCD:-no}
 
+DISTRO_IDS=$(awk -F= '/^ID/{print$2}' /etc/os-release | tr -d '"')
```

Comment:
This works now but still sets `DISTRO_IDS` to `rhel fedora` on my RHEL machine.

Nit: I think I'd still change this to:
 ```
DISTRO_IDS=$(awk -F= '/^ID=/{print$2}' /etc/os-release | tr -d '"')
```

But I'll leave it up to you, it's OK as is too.

Reply:
Using the `^ID` to get both the ID and ID_LIKE fields was intentional, that allows the check functions to iterate over all the ID's to help them decide what kind of distro we're dealing with. I'll add a comment to make that more clear.

Reply:
Ok, but I'm not sure I understand where the iteration is.  Did you mean something like:
```
function is_rpm_based() {
    for i in $DISTRO_IDS; do
        case $i in
        centos* | rhel* | fedora*)
            true
            return
        ;;
        esac
    done
    false
}
```

Reply:
So, I intuitively assumed the `in` operator would do both, and AFAICT it does?

```bash
#!/bin/bash

function is_abc() {
    echo "evaluatiing: '$DISTRO_IDS'"
    case $DISTRO_IDS in
        a* | b* | c*)
            echo a, b or c
        ;;
        *)
            echo neither
        ;;
    esac	
}

DISTRO_IDS="c"
is_abc
DISTRO_IDS="c b"
is_abc
DISTRO_IDS="c b
a"
is_abc
DISTRO_IDS="a b
c"
is_abc
DISTRO_IDS="d e
f"
is_abc
```

Output:
```
evaluatiing: 'c'
a, b or c
evaluatiing: 'c b'
a, b or c
evaluatiing: 'c b
a'
a, b or c
evaluatiing: 'a b
c'
a, b or c
evaluatiing: 'd e
f'
neither
```

Reply:
Done

Reply:
Thanks!